styles/settings/_typography.scss

```diff
@@ -6,6 +6,7 @@ $typo-stack-base: 'Whitney SSm A', 'Whitney SSm B', sans-serif;
 /* ==========================================================================
    Sizes
    ========================================================================== */
+$typo-size-header: 1.4rem;
```

Comment:
This feels like a component setting rather than a global one. What do you think?